cli/cmd/upgrade_test.go

```diff
@@ -597,7 +597,7 @@ func TestUpgradeOverwriteRemoveAddonKeys(t *testing.T) {
 
 	install := renderInstall(t, installOpts)
 
-	flagSet.Set("addon-config", filepath.Join("testdata", "grafana_enabled.yaml"))
+	flagSet.Set("config", filepath.Join("testdata", "grafana_enabled.yaml"))
 	flagSet.Set("addon-override", "true")
```

Comment:
The flag is called `addon-overwrite` but this references `addon-override`. Are these things different?

Reply:
Good catch.  Will fix in a separate PR.